### DISCUSSION TITLE
Override styles for daisy-ui class inline?

### DISCUSSION THREAD

**Discussion by lsbyerley** [saadeghi/daisyui#761] [Q&A] [answered]
I've noticed if I need to modify the specifics of a daisy-ui class such as `card-title` by adding a `mb-0`, the margin-bottom is overridden by daisy-ui. This is sometimes just a one-off case. Is there a way to do so without overriding the class everywhere in the config?

**Comment by saadeghi** [comment, marked as answer]
`card-title` has no margin by default so setting the margin to `0` using `mb-0` does nothing (because it is `0` already)  

However, `card-body` is a flex layout with `0.5rem` gap.  
This adds a gap between all elements inside `card-body`.  
If you want to remove that gap, add `gap-0` to `card-body`
